apps/api/src/campaign/campaign.service.ts

```diff
@@ -408,10 +410,25 @@ export class CampaignService {
         { create: { campaignId, currency: campaign.currency, name: campaign.title } }
 
     // Find donation by extPaymentIntentId and update if status allows
+    const donationNotificationSelect = {
+      id: true,
+      status: true,
+      currency: true,
+      amount: true,
+      createdAt: true,
+      extPaymentMethodId: true,
+      person: {
+        select: {
+          firstName: true,
+          lastName: true,
+          picture: true,
+        },
+      },
```

Comment:
@AndreyGoranov Is it expected to expose the data of the person who donated to all other website visitors?

We should restrict the personal data in the notifications based on the user's choice.

The user can select to stay anonymous when they donate, so we shouldn't expose that info to the general public.

cc @igoychev @dimitur2204

Reply:
How can I check if the user selected to stay Anonymous?

Reply:
@AndreyGoranov I've just researched the topic and it seems that the `personId` of the user that selected `anonymous` is never sent to the backend, so I assume that the donations with a connected person are always non-anonymous.

Having that in mind I'll resolve the discussion.

Reply:
yes, when the user wants to donate anonymously, the donation record will not be connected to a person entity. So for anonymous donations, only the billingName of the donation will hold the name, and we don't display that publicly.